evm/src/main/java/io/horizen/evm/TraceOptions.java

```diff
@@ -0,0 +1,46 @@
+package io.horizen.evm;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class TraceOptions {
```

Comment:
`timeout` field was removed during the refactoring.
why?

Reply:
It was never used and already removed on the go side in the original PR. On the SDK side it was kept accidentally.

Reply:
ok, I see.
was confused with the outdated reference: https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#traceconfig